modVM/src/org/aion/vm/Forks.java

```diff
@@ -0,0 +1,22 @@
+package org.aion.vm;
+
+public class Forks {
+
+    private static final long SEPTEMBER_FORK = 1000000L;
```

Comment:
Maybe name it SEPTEMBER_2018 fork... in case we have another fork after that, that's also in September :)